addons/ccm-vsphere/ccm-vsphere.yaml

```diff
@@ -163,30 +163,29 @@ spec:
         "kubeone.k8c.io/cabundle-hash": "{{ .Config.CABundle | sha256sum }}"
         "kubeone.k8c.io/cloudconfig-hash": "{{ .Config.CloudProvider.CloudConfig | sha256sum }}"
     spec:
-      priorityClassName: system-cluster-critical
       nodeSelector:
         node-role.kubernetes.io/control-plane: ""
-      securityContext:
-        runAsUser: 1001
       tolerations:
         - key: node.cloudprovider.kubernetes.io/uninitialized
           value: "true"
           effect: NoSchedule
-        - key: node-role.kubernetes.io/control-plane
+        - key: node-role.kubernetes.io/master
```

Comment:
Is this correct?

Reply:
Yes. Upstream manifest switched the order of tolerations for some reason, but that doesn't really change anything.